### DISCUSSION TITLE
Custom Autopilot

### DISCUSSION THREAD

**Discussion by Zimmerbz** [flybywiresim/aircraft#3022] [General]
Just curious, how far along are you with the custom autopilot, in particular managed climb and descent?  I see the the TRK/FPA is implemented in some of the recent dev versions but curious as to how far things are along.

**Comment by Inaxair** [comment]
Hello @Zimmerbz 

We are making progress, we have Autoland coming, custom VNAV, new systems also coming! Just be patient, but don't worry, it will come soon!

Best regards,
Inaxair from FlyByWire Simulations
